vagrant/provisioning/roles/microservices/tasks/main.yaml

```diff
@@ -16,3 +16,53 @@
     block: "{{ lookup('template', 'templates/ssl.conf') }}"
     marker: "# {mark} ANSIBLE MANAGED BLOCK for microservices"
     insertbefore: "BrowserMatch"
+
+# CentOS 7 ships with HTTPD version 2.4.6. We need at least version 2.4.11 for the SubstituteMaxLineLength directive
```

Comment:
Why do we need to store zipkin-server jar in the repository, instead of downloading it during installation?

Reply:
We can do that. Thanks.